docs/source/contributing/developing.adoc

```diff
@@ -316,6 +316,13 @@ The following options are available:
 
 Tags::
 Use `tags` to ensure that scenarios and features containing at least one of the selected tags are executed.
+To select particular feature, you can use its name as a tag.
+For example, the `cmd-config.feature` contains `@cmd-config` tag through which it can be selected and run with the following command: `make integration GODOG_OPTS=--tags=cmd-config`.
```

Comment:
```asciidoc
To select particular feature, you can use its name as a tag.
For example, the `cmd-config.feature` contains `@cmd-config` tag through which it can be selected and run with the following command: `make integration GODOG_OPTS=--tags=cmd-config`.
```
This is clearer now.. :+1: